activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   `AR::Relation#reverse_order` throws `AR::IrreversibleOrderError` when the order can not be reversed
```

Comment:
Since we have no `ActiveResource` there is no doubt what is `AR`. Spelling it makes lines too long.